### PR TITLE
Refactor

### LINKED ISSUE #196: Add practice repo tests

### LINKED ISSUE #258: Remove speech recognition
There is absolutely dead code. No way to use it in the future. Just confusing island of trash.

### PR DESCRIPTION
**Pull request by winter-yuki**
Closes #258 
Closes #196 

Depends on #252

### CODE REVIEW

**Comment by zuevval** [comment]
maybe move `database.md`, etc to `doc` directory?

File: app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/practice/DecksListFragment.kt [hunk 1]
```diff
@@ -58,6 +60,11 @@ class DecksListFragment : Fragment() {
             decksList.adapter = DecksListAdapter(decks) { item ->
                 this.item = item
                 deckName.text = deckTagToName.getValue(item.deck.tag)
+                deckName.setTypeface(
+                    deckName.typeface,
+                    if (item.deck == currDeck) Typeface.BOLD
+                    else Typeface.NORMAL
+                )
```
Comment (zuevval) [inline app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/practice/DecksListFragment.kt:67]: This is the most valuable change in this PR, thanks

Comment (zuevval) [reply app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/practice/DecksListFragment.kt:67]: At least for user

File: app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/menu/MenuFragment.kt [hunk 2]
```diff
@@ -125,28 +121,6 @@ class MenuFragment : AbstractFragmentWithHelp(R.string.menu_help) {
         }
     }
 
-    override fun onRequestPermissionsResult(
```
Comment (zuevval) [inline app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/menu/MenuFragment.kt:128]: R. I. P. sound recognition. Maybe later we resurrect you.

**Review by zuevval — APPROVED** [review]
I've checked and tested. It's OK

File: app/src/androidTest/java/com/github/braillesystems/learnbraille/res/ResourceTest.kt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ResourceTest {
+
+    private lateinit var context: Context
+
+    @Before
+    fun before() {
```
[the remark this answers is not in the document]
Comment (winter-yuki) [reply app/src/androidTest/java/com/github/braillesystems/learnbraille/res/ResourceTest.kt]: Yes, thank you

**Pull request merged**